unox.py

```diff
@@ -20,14 +20,20 @@
 
 import sys
 import os
-import time
 import fsevents
-import urllib
 import traceback
 import signal
 
+# Import depending on python version
+if sys.version_info.major < 3:
+    from urllib import quote, unquote
+else:
+    from urllib.parse import quote, unquote
```

Comment:
Jup. I've tested it manually.

I've started the script in a python2 and another python3 virtualenv.

I've also changed the first line to `#!/usr/bin/env python2` and `#!/usr/bin/env python3` and it detects filesystem changes in my docker-sync setup. 

😃